build/README.md

```diff
@@ -574,8 +574,10 @@ Compile the ping binary and then build the docker image
 #### `make gen-install`
 Generate the `/install/yaml/install.yaml` from the Helm template
 
-#### `gen-embedded-openapi`
-Generate the embedded OpenAPI specs for existing Kubernetes Objects, such as `PodTemplateSpec` and `ObjectMeta`
+#### `make gen-embedded-openapi`
+Generate the embedded OpenAPI specs for existing Kubernetes Objects, such as `PodTemplateSpec` and `ObjectMeta`.
+
+This should be run against a clean or brand new cluster, as external CRD's or schemas could cause errors to occur.
```

Comment:
Is there a build target to do this? I found that when I created clusters manually that the kubectl proxy command in the script couldn't find them but when I created a cluster with `make gcloud-test-cluster ` it was able to connect.

Reply:
`make gcloud-test-cluster` does work - although admittedly that will install Agones (which I found out the hard way, I did a `helm uninstall` on it) - so there isn't a make target that makes an empty cluster.

Reply:
Thanks. I've updated the upgrade checklist to include the helm uninstall command.